apollos/core/components/header/index.jsx

```diff
@@ -0,0 +1,120 @@
+
+import { Component, PropTypes } from "react"
+import { connect } from "react-redux"
+
+@connect((state) => ({
+  color: state.header.content.color || "#6BAC43",
+  light: state.header.content.light,
+  text: state.header.content.title,
+  subText: state.header.content.subTitle,
+  visible: state.header.visible,
+  isSearch: state.header.content.isSearch,
+  searchSubmit: state.header.content.searchSubmit
+}))
+export default class Header extends Component {
+
+  render () {
+    const lightColor = "text-light-primary";
+    const darkColor = "text-dark-primary";
+
+    let text = lightColor;
+    if (!this.props.light) {
+      text = darkColor;
+    }
+
+    if (!this.props.visible) {
+      return null;
+    }
+
+    let paddingTop = "15px";
+
+    if(this.props.isSearch) {
+      paddingTop = "0";
+    }
+
+    return (
+      <div class="hard">
```

Comment:
@benwiley this should be `className`, and may not be necessary since it currently doesn't get rendered.